main.php

```diff
@@ -167,9 +180,11 @@ function tc_form() {
 		echo ( basename( STYLESHEETPATH ) === $location['Stylesheet'] ) ? 'value="' . $location['Stylesheet'] . '" style="font-weight:bold;">' . $name . '</option>' : 'value="' . $location['Stylesheet'] . '">' . $name . '</option>';
 	}
 	echo '</select>';
-	echo '<input class="button" type="submit" value="' . __( 'Check it!', 'theme-check' ) . '" />';
-	if ( defined( 'TC_PRE' ) || defined( 'TC_POST' ) ) echo ' <input name="trac" type="checkbox" /> ' . __( 'Output in Trac format.', 'theme-check' );
-	echo '<input name="s_info" type="checkbox" /> ' . __( 'Suppress INFO.', 'theme-check' );
+	echo '<input class="button" type="submit" value="' . esc_html__( 'Check it!', 'theme-check' ) . '" />';
```

Comment:
This would need be esc_attr, since it's inside an attribute.

Reply:
```suggestion
	echo '<input class="button" type="submit" value="' . esc_attr__( 'Check it!', 'theme-check' ) . '" />';
```